src/Symfony/Component/Mailer/Bridge/Mailjet/Transport/MailjetApiTransport.php

```diff
@@ -151,9 +153,15 @@ private function getPayload(Email $email, Envelope $envelope): array
             $message['Headers'][$header->getName()] = $header->getBodyAsString();
         }
 
-        return [
+        $returnArray = [
             'Messages' => [$message],
         ];
+
+        if ($this->sandbox) {
+            $returnArray['SandboxMode'] = true;
+        }
```

Comment:
You could consider setting the `SandboxMode` key in any case like:
```
return ['Messages' => [$message], 'SandBoxMode => $this->sandbox];
```
should clean up this function a bit in complexity and should make debugging slightly easier by being explicit about the value